backend/src/services/reviewed-cars/reviewed-cars.service.ts

```diff
@@ -0,0 +1,94 @@
+import { prisma } from '@data/prisma-client';
+
+import type {
+  setViewedCarRequest,
+  setViewedCarResponse,
+} from '@autoline/shared';
+import type { getViewedCarsListResponse } from '@common/types/types';
+
+const getList = async (
+  userId: string,
+): Promise<(getViewedCarsListResponse | null | undefined)[]> => {
+  const viewedСarsList = await prisma.users_Viewed_Cars.findMany({
+    where: {
+      user_id: userId,
+    },
+    orderBy: {
+      created_at: 'desc',
+    },
+    select: {
+      model_id: true,
+      complectation_id: true,
+    },
+  });
+
+  const detailedListOfReviewedCars = await Promise.all(
+    viewedСarsList.map(async ({ model_id, complectation_id }) => {
+      if (model_id && complectation_id) {
+        return await prisma.model.findFirst({
```

Comment:
`return await` is redundant. Only `return` can be used, the calling side will await

Reply:
Done